### PR TITLE
Add v to front of version tag and cleanup

### PR DESCRIPTION
**Pull request by onyiny-ang**

### CODE REVIEW

File: scripts/imagebuild.sh [hunk 1]
```diff
@@ -49,7 +49,7 @@ docker login -u "${QUAY_USERNAME}" -p "${QUAY_PASSWORD}" quay.io
 echo "Building Federation-v2 docker image"
 docker build ${dockerfile_dir} -t ${REGISTRY}${REPO}/federation-v2:${TAG}
 
-echo "Pushing images with default tags (git sha and 'canary')."
+echo "Pushing images with tag '${TRAVIS_TAG}'."
```
Comment (pmorie) [inline scripts/imagebuild.sh]: image

File: scripts/imagebuild.sh [hunk 2]
```diff
@@ -25,11 +25,11 @@ dockerfile_dir="${base_dir}/images/federation-v2"
 
 echo "travis tag: ${TRAVIS_TAG}"
 echo "travis branch:${TRAVIS_BRANCH}"
-if [[ "${TRAVIS_TAG}" =~ ^[0-9]+\.[0-9]+\.[0-9]?\d+([-a-zA-Z0-9-]*)? ]]; then
-    echo "Pushing images with tag '${TRAVIS_TAG}'."
+if [[ "${TRAVIS_TAG}" =~ ^v([0-9]\.)+([0-9])[-a-zA-Z0-9]*([.0-9])* ]]; then
+   echo "Using tag: '${TRAVIS_TAG}'."
```
Comment (pmorie) [inline scripts/imagebuild.sh]: Might want to say `Using image tag`, since there are a couple different types of tags this could refer to

Comment (onyiny-ang) [reply scripts/imagebuild.sh]: Yes you're right.

File: scripts/imagebuild.sh [hunk 3]
```diff
@@ -25,11 +25,12 @@ dockerfile_dir="${base_dir}/images/federation-v2"
 
 echo "travis tag: ${TRAVIS_TAG}"
 echo "travis branch:${TRAVIS_BRANCH}"
-if [[ "${TRAVIS_TAG}" =~ ^[0-9]+\.[0-9]+\.[0-9]?\d+([-a-zA-Z0-9-]*)? ]]; then
-    echo "Pushing images with tag '${TRAVIS_TAG}'."
+if [[ "${TRAVIS_TAG}" =~ ^v([0-9]\.)+([0-9])[-a-zA-Z0-9]*([.0-9])* ]]; then
```
Comment (pmorie) [inline scripts/imagebuild.sh]: Something changed about this regex - it's now looking for `1.1-blah` not `1.1.1-blah`.

**Pull request merged**